### ISSUE TITLE
Videos aren't being played

### ISSUE THREAD

**Issue by pedrohff** [lmartinking/webscreensaver#13]
First time I opened something with webscreensaver it said I had no webkit installed, or something like "namespace webkit not available", message was in yellow. Searching a bit, I found I needed this: gir1.2-webkit-3.0. Installed it, worked fine until I tried to play a video. The video works in Chrome and Firefox, but not in the webscreensaver. 
All I know for now is that it uses this user-agent, not sure if this can help someone to help me:
Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/538.15 (KHTML, like Gecko) Version/8.0 Safari/538.15
  
Also I'm using this HTML+JS https://github.com/archagy/webnuxsaver, should I post this issue there as well?

**Comment by lmartinking** [comment]
Hi there, I think this might be due to the video format? It really depends on what gtk webkit supports in terms of codecs -- it's not exactly the same as Chrome or Firefox, unfortunately!

**Comment by smos** [comment]
Using "sudo apt-get -qy install ubuntu-restricted-addons gstreamer1.0-plugins-ugly" I managed to get it working for the Raspberry Pi and on a Ubuntu PC.

I do however run into another issue using Video. It keeps hanging in the process list after a signal churning ~40% CPU. This is also reported by xscreensaver that the previous process did not quit untill the video stops looping.

```
xscreensaver: 15:36:12: 0: destroyed old saver window 0x1200052.
xscreensaver: 15:36:12: WARNING: pid 6385 (webscreensaver) sent SIGTERM 67 seconds ago and did not die!
```
I do see other references to Gtk.main_quit() not working as expected, but unfamiliarity with Python code prevents me from fixing this myself. I added a os.system() under terminate, but it fails to reach that too.
